features/configuration/run_all_when_everything_filtered.feature

```diff
@@ -1,78 +0,0 @@
-Feature: run all when everything filtered
```

Comment:
In the vein of the conversation about deprecation, I think we should leave this in, but mark it as deprecated in bold at the top...